scripts/bump.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+const standardVersion = require('standard-version');
+const fs = require('fs');
+const path = require('path');
+const ver = require('./resolve-version');
+const repoRoot = path.join(__dirname, '..');
+
+const releaseAs = process.argv[2] || 'minor';
+const forTesting = process.env.BUMP_CANDIDATE || false;
+
+async function main() {
+  if (releaseAs !== 'minor' && releaseAs !== 'patch') {
+    throw new error(`invalid bump type "${releaseAs}". only "minor" (the default) and "patch" are allowed. major version bumps require *slightly* more intention`);
+  }
+
+  console.error(`Starting ${releaseAs} version bump`);
+  console.error('Version information:', JSON.stringify(ver, undefined, 2));
+
+  const changelogPath = path.join(repoRoot, ver.changelogFile);
+  const opts = {
+    releaseAs: releaseAs,
+    skip: { tag: true },
+    packageFiles: [ { filename: ver.versionFile, type: 'json' } ],
+    bumpFiles: [ { filename: ver.versionFile, type: 'json' } ],
+    infile: ver.changelogFile,
+    prerelease: ver.prerelease,
```

Comment:
Will this correctly update pre-release versioning? Specifically, will `2.0.0-alpha.1` go to `2.0.0-alpha.2`.

This is not clear from skimming through the [docs](https://github.com/conventional-changelog/standard-version#release-as-a-pre-release)

Reply:
Yes, that's what the `prerelease` feature in standard-version does.